docs/src/pages/guides/minimizing-bundle-size/minimizing-bundle-size.md

```diff
@@ -168,6 +168,7 @@ Create a `config-overrides.js` file in the root directory:
 
 ```js
 /* config-overrides.js */
+/* eslint-disable react-hooks/rules-of-hooks */
 const { useBabelRc, override } = require('customize-cra');
```

Comment:
How is the disable directive helping here? The import alone doesn't trigger any violation as far as I know. Only the callsite would issue the error.

The issue even had a repro that confirms this:

> red warning override(useBabelRC())

Reply:
Yeah, to be fair, I think that we could have use: `// eslint-disable-next-line react-hooks/rules-of-hooks` just above of the usage of the import. But 🤷‍♂️